jmix-flowui/flowui/src/main/java/io/jmix/flowui/UiComponentProperties.java

```diff
@@ -112,6 +114,11 @@ public class UiComponentProperties {
      */
     boolean showErrorMessageBelowField;
 
+    /**
+     * Whether to trim the entered string by default for {@link HasTrimming} components.
+     */
+    boolean defaultTrimEnabled;
```

Comment:
we discussed that this feature should be enabled by default and disabled during migration process

Reply:
Sure, the default value is `true` by default (see the `UiComponentProperties` class constructor)
The issue for the studio will be created after the merging (as stated in the PR description)

Reply:
Yeah, missed that :)